src/utils/SSTORE2.sol

```diff
@@ -79,7 +79,7 @@ library SSTORE2 {
         uint256 start,
         uint256 size
     ) private view returns (bytes memory data) {
-        assembly {
+        assembly ("memory-safe") {
```

Comment:
There can be dirty values at the end of the byte string btw.

Need to clean the last memory chunk here.

Take size to be 31 and have all the memory to be dirty. The last bit of the `data` would be dirty.

Reply:
Yes, that's correct, but shouldn't affect `memory-safe`?

Reply:
Yeah. This is independently of memory safe. I'll separate it into a new issue.